implementations/rust/ockam/ockam_command/src/environment/static/env_info.txt

```diff
@@ -21,6 +21,20 @@ Logging
 - OCKAM_LOG_MAX_FILES: an `integer` that defines the maximum number of log files to keep per node. Default value `60`.
 - OCKAM_LOG_CRATES_FILTER: a filter for log messages based on crate names: `all`, `default`, comma-separated list of crate names. Default value: `default`, i.e. the list of `ockam` crates.
 
+Database
```

Comment:
Didn't know we have this file :D

Reply:
Yeah, it's actually the output of the `ockam environment` command